### ISSUE TITLE
Tensorflow loss increase during the training

### ISSUE THREAD

**Issue by abdou31** [tensorflow/tensorflow#28822]
I cloned a github project that using tensorflow :
+ https://github.com/yinguobing/cnn-facial-landmark

and i tried to training ibug dataset (300VW , 300W , Helen...) to get a trained tensorflow model 
but when training starting , the result is good and the loss decreasing during training , but after 1000 steps , the loss value increase and decrease during the training , I don't have any idea about this changing of loss value is a problem or bug or not , 
How can i fix that?

![image](https://user-images.githubusercontent.com/19480228/57968022-700bf380-7965-11e9-8256-e130e12c5d97.png)

**System information**
- The source code : https://github.com/yinguobing/cnn-facial-landmark/blob/master/landmark.py
- OS Platform and Distribution : Windows 10 x64
- TensorFlow installed from : binary
- TensorFlow version : 13.1.1 ( GPU )
- Python version: 3.6
- CUDA/cuDNN version:10.0/7.4
- GPU model and memory:Nvidia Geforce 840m 4 Go

**Comment by Transwert** [comment]
might be due to inconsistency in data set, check through it once,

**Comment by abdou31** [comment]
i have ibug dataset that contains ( 300VW 300W , Helen..) => images and .json files (annotation)

**Comment by abdou31** [comment]
I probarly making a false steps of dataset organization , I have generated test.record , train.record and valid.record with 300VW image and the image dataset contains 300VW , 300W , helen ...
Can this give me errors?

**Comment by jvishnuvardhan** [comment]
This is not Build/Installation or Bug/Performance issue. Please post this kind of support questions at [Stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow). There is a big community to support and learn from your questions. GitHub is mainly for addressing bugs in installation and performance. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by abdou31** [comment]
Yes , the problem is in the loss function but i didn't know how can i solve it?

**Comment by jvishnuvardhan** [comment]
I am closing this issue as it is not related to Build/install or Bug/performance. 
Please post this kind of support questions at Stackoverflow. There is a big community to support and learn from your questions. GitHub is mainly for addressing bugs in installation and performance. Thanks!
